provision/acc_provision/templates/aci-containers.yaml

```diff
@@ -2202,6 +2202,21 @@ spec:
         - name: aci-containers-host
           image: {{ config.registry.image_prefix }}/aci-containers-host:{{ config.registry.aci_containers_host_version }}
           imagePullPolicy: {{ config.kube_config.image_pull_policy }}
+          {% if config.kube_config.aci_containers_host_memory_limit or config.kube_config.aci_containers_host_memory_request %}
+          resources:
+            limits:
+              {% if config.kube_config.aci_containers_host_memory_limit %}
+              memory: "{{ config.kube_config.aci_containers_host_memory_limit }}"
+              {% else %}
+              memory: "{{ config.kube_config.aci_containers_memory_limit }}"
+              {% endif %}
+            requests:
+              {% if config.kube_config.aci_containers_host_memory_request %}
+              memory: "{{ config.kube_config.aci_containers_host_memory_request }}"
+              {% else %}
+              memory: "{{ config.kube_config.aci_containers_memory_request }}"
+              {% endif %}
+          {% endif %}
```

Comment:
do we really need this additional if checks, namespace wide request and limit will automatically apply right??

Reply:
Memory request value is not picked from namespaced resource, it is set same as container memory limit.
(this was the issue with ovs_memory_limit and we had to explicitly add new flag ovs_memory_request)